ext/SymbolicsLuxExt.jl

```diff
@@ -11,9 +12,13 @@ using Symbolics.SymbolicUtils
     end
 end
 
+function LuxCore.outputsize(model::SymbolicUtils.BasicSymbolic{<:LuxCore.AbstractLuxLayer}, x::Symbolics.Arr, rng::AbstractRNG)
+    LuxCore.outputsize(Symbolics.getmetadata(model, Symbolics.VariableDefaultValue), x, rng)
```

Comment:
```suggestion
    LuxCore.outputsize(Symbolics.getdefaultval(model), x, rng)
```

Reply:
thanks, I couldn't remember the name for this one :sweat_smile: